packages/strapi-admin/files/public/app/components/LeftMenuLinkContainer/index.js

```diff
@@ -5,51 +5,82 @@
  */
 
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 import LeftMenuLink from 'components/LeftMenuLink';
+
 import styles from './styles.scss';
+import messages from './messages.json';
 
 class LeftMenuLinkContainer extends React.Component { // eslint-disable-line react/prefer-stateless-function
   render() {
+    // Generate the list of sections
+    const linkSections = this.props.plugins.valueSeq().map(plugin => (
+      plugin.get('leftMenuSections').map((leftMenuSection, j) => {
+        const sectionlinks = leftMenuSection.get('links').map((sectionLink, k) => (
+          <LeftMenuLink
+            key={k}
+            icon={sectionLink.get('icon') || 'link'}
+            label={sectionLink.get('label')}
+            destination={`/plugins/${plugin.get('id')}/${sectionLink.get('destination')}`}
+          />
+        ));
+
+        return (
+          <div key={j}>
+            <p className={styles.title}>{leftMenuSection.get('name')}</p>
+            <ul className={styles.list}>
+              {sectionlinks}
+            </ul>
+          </div>
+        );
+      })
+    ));
+
+
     // List of links
-    let links = this.props.plugins.valueSeq().map((plugin) => (
+    let pluginsLinks = this.props.plugins.valueSeq().map((plugin) => (
       <LeftMenuLink
         key={plugin.get('id')}
         icon={plugin.get('icon') || 'plug'}
         label={plugin.get('name')}
         destination={`/plugins/${plugin.get('id')}`}
-        leftMenuLinks={plugin.get('leftMenuLinks')}
       />
     ));
 
     // Check if the plugins list is empty or not
-    if (!links.size) {
-      links = <span className={styles.noPluginsInstalled}>No plugins installed yet.</span>;
+    if (!pluginsLinks.size) {
```

Comment:
This block should be before the mapping. It will avoid a `map` call and to reapply a value to the `pluginsLinks` variable, then you use a `const` instead of `let`.